.ci/pipelines/e2e-tests-ocp-all-but-latest.Jenkinsfile

```
@@ -0,0 +1,90 @@
+// This library overrides the default checkout behavior to enable sleep+retries if there are errors
+// Added to help overcome some recurring github connection issues
+@Library('apm@current') _
+
+def failedTests = []
+def lib
+
+pipeline {
+
+    agent {
+        label 'linux'
+    }
+
+    options {
+        timeout(time: 50, unit: 'HOURS')
+    }
+
+    environment {
+        VAULT_ADDR = credentials('vault-addr')
+        VAULT_ROLE_ID = credentials('vault-role-id')
+        VAULT_SECRET_ID = credentials('vault-secret-id')
+        GCLOUD_PROJECT = credentials('k8s-operators-gcloud-project')
+    }
+
+    stages {
+        stage('Load common scripts') {
+            steps {
+                script {
+                    lib = load ".ci/common/tests.groovy"
+                }
+            }
+        }
+        stage("Run E2E tests") {
+            when {
+               expression {
+                    // this is a downstream job but let's not run it every day due to the amount of resources
+                    // it requires
+                    isFriday()
+               }
+            }
+            steps {
+                // "4.3.40", "4.4.33", "4.5.37", "4.6.24", "4.7.6"
```

Nit: for future updates
```suggestion
                // "4.3.x", "4.4.x", "4.5.x", "4.6.x", "4.7.x"
```